practice_app/api/urls.py

```diff
@@ -4,5 +4,6 @@
 app_name = "api"
 urlpatterns = [
     path("google-scholar/", views.google_scholar, name="google-scholar"),
-    path('eric/', views.eric_papers, name='eric_papers')
+    path('eric/', views.eric_papers, name='eric_papers'),
+    path('semantic-scholar/', views.semantic_scholar, name='semantic-scholarc')
```

Comment:
I guess there is a typo in "name= 'semantic-scholarc' ". Name might be 'semantic-scholar' to avoid conflicts in future.

Reply:
ok. i fixed it.